.pre-commit-config.yaml

```diff
@@ -25,3 +26,8 @@ repos:
         files: ^assets/javascript/.*\.[jt]sx?$  # *.js, *.jsx, *.ts and *.tsx
         types: [ file ]
         args: [ --fix ]
+  - repo: https://github.com/adamchainz/djade-pre-commit
```

Comment:
I'm not sure if ordering matters but maybe put this above djhtml since it might fix issues that djhtml would stumble on?